packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -39,8 +35,11 @@ export interface PanelChromeProps {
   children: (innerWidth: number, innerHeight: number) => ReactNode;
   padding?: PanelPadding;
   title?: string;
-  titleItems?: PanelChromeInfoState[];
-  menu?: ReactElement;
+  description?: string | (() => string);
+  links?: () => Array<LinkModel<PanelModel>>;
+  panelNotices?: Notices;
```

Comment:
I don't think this interface makes much sense, think a simple react node | function return react node is better here (title items should suffice )